web/modules/custom/mymodule/mymodule.routing.yml

```diff
@@ -0,0 +1,23 @@
+mymodule.simplecontent:
+  path: '/mycustompath'
+  defaults: 
+    _controller: 'Drupal\mymodule\Controller\FirstController::simpleContent'
+    _title: 'Custom Title 1'
```

Comment:
Please provide proper values for titles.

Reply:
Provided